src/test/java/de/tum/in/www1/artemis/util/TimeService.java

```diff
@@ -0,0 +1,23 @@
+package de.tum.in.www1.artemis.util;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class TimeService {
```

Comment:
There already are utility methods for converting time units in Java.
E.g. nanos to millis:
`TimeUnit.NANOSECONDS.convert(Duration.ofMillis(millSeconds))`

Reply:
Thanks! I'll change it.